_pytest/main.py

```diff
@@ -175,7 +175,7 @@ def pytest_ignore_collect(path, config):
     if excludeopt:
         ignore_paths.extend([py.path.local(x) for x in excludeopt])
 
-    if path in ignore_paths:
+    if py.path.local(path) in ignore_paths:
```

Comment:
i propose either fixing pylib and updating the requirement or  adding a comment for the normalization so we understand why it happens and what issue in pylib causes it